PowerTasks/.Tasks/OctoCreateRelease.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+	<metadata>
+		<id>PowerTasks.Plugins.OctoCreateRelease</id>
+		<version>1.0.0</version>
+		<authors>Shayne van Asperen, Kenneth Truyers</authors>
+		<owners>Shayne van Asperen</owners>
+		<licenseUrl>https://github.com/shaynevanasperen/PowerTasks/blob/master/LICENSE</licenseUrl>
+		<projectUrl>https://github.com/shaynevanasperen/PowerTasks</projectUrl>
+		<iconUrl>https://raw.githubusercontent.com/shaynevanasperen/PowerTasks/master/PowerTasks.png</iconUrl>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<summary>A PowerTasks plugin which creates a release on an Octopus Deploy server and optionally deploys it</summary>
+		<description>OctoCreateRelease</description>
+		<language>en-US</language>
+		<tags>powershell, invoke, build, tasks, plugin, deploy, octopus</tags>
+		<developmentDependency>true</developmentDependency>
+		<dependencies>
+			<dependency id="PowerTasks" version="2.0.0" />

```

Comment:
You should add a dependency on OctopusTools here:
`<dependency id="OctopusTools" version="3.2.0.91" />`